lib/api/core/models/repositories/roleRepository.js

```diff
@@ -130,6 +130,19 @@ class RoleRepository extends Repository {
       });
   }
 
+  /**
+   * @override
+   */
+  loadOneFromDatabase (id) {
```

Comment:
Without this override, the error returned in case of a 404 would be a generic `services.storage.not_found`